src/xdocs/config_coding.xml

```diff
@@ -941,11 +941,37 @@ String nullString = null;
 
       <subsection name="Examples" id="EqualsAvoidNull_Examples">
         <p>
-          To configure the check:
+            To configure the check:
         </p>
         <source>
 &lt;module name=&quot;EqualsAvoidNull&quot;/&gt;
         </source>
+          <p>
```

Comment:
indentation here and below
For tags - they all should be on same level (now indented by 2 spaces)
For text in tags (except code examples) - 2 spaces indentation